src/plugins_exts.c

```diff
@@ -403,6 +403,131 @@ lyplg_ext_print_get_level(const struct lyspr_ctx *ctx)
     return &((struct lyspr_ctx *)ctx)->level;
 }
 
+LIBYANG_API_DECL LY_ERR
+lyplg_ext_sprinter_ctree_add_ext_nodes(const struct lyspr_tree_ctx *ctx, struct lysc_ext_instance *ext,
+        lyplg_ext_sprinter_ctree_override_clb clb)
+{
+    LY_ERR rc = LY_SUCCESS;
+    uint32_t i;
+    struct lysc_node *schema;
+
+    assert(ctx);
```

Comment:
You cannot put `assert()` on a param in a public function. Use `LOGARG` macro. Seen several times.